build/dependencies.txt

```diff
@@ -199,6 +201,8 @@
 ../src/framework/utils/entity-reference.js
 ../src/framework/utils/sorted-loop-array.js
 ../src/resources/loader.js
+../src/resources/bundle.js
+../src/resources/untar.js
```

Comment:
This library is not currently included.

Reply:
Grouping multiple assets into large blobs is definitely a good idea. I wonder if you can just reuse existing browser GZIP compression and simplify the task into joining/splitting data without any fancy logic? E.g. you only need a file table with byte offsets + large chunk of glued data.